cli/turbo.json

```diff
@@ -3,7 +3,34 @@
   "extends": ["//"],
   "pipeline": {
     "build": {
-      "outputs": ["turbo", "turbo.exe"]
+      "outputs": [
+        // TODO: this gets moved over to ../target, can we omit this?
+        "turbo",
+        // TODO: does this get moved over to ../target also? Can we omit this too?
+        "turbo.exe",
+        // TODO: This is 7GB, do we want to ever even cache the debug build?
+        "../target/debug/**",
+        // TODO: Do we need the whole directory? go-turbo and turbo are ~45mb, the whole directory is 800mb
+        "../target/release/**"
+      ],
+
+      "inputs": [
+        "**/*.go",
+        "**/*_test.go",
+
+        // Rust crates that should trigger a rebuild
+        // TODO: this is not the full list (e.g. pidlock is missing here)
+        // what else do we need here? And should turbo be crawling / using cargo
+        // to figure this out automatically? Or should we wire up a package.json dependency graph?
+        "../crates/turborepo/**",
+        "../crates/turbopath/**",
+        "../crates/turborepo-*/**",
+
+        // Exclude target directories of everything specified above
+        "!../crates/turborepo/target/**",
```

Comment:
These shouldn't be here unless explicitly specified as an output directory (like in the `turborepo-ffi` rule in the Makefile). Otherwise the default output is `target/` at the root of the repo.

Reply:
Just making sure you see that these are `!` prefixed?  (which means that it's configured to "shouldn't be here") as you say?

Reply:
Yeah, so it's nbd either way, but mostly point out that these directories won't exist in the first place

Reply:
oh, I only added them because they did exist 🤔 , otherwise wouldn't have known to exclude them. I'll double check